src/stories/standalone/UIKit/InputControls/PhoneNumberInput.story.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { boolean, text, select } from "@storybook/addon-knobs";
+import PhoneNumberInput from "../../../../standalone/UIKit/InputControls/PhoneNumberInput";
+import { CountryCode } from "libphonenumber-js";
+import { action } from "@storybook/addon-actions";
+
+export const PhoneNumberInputStory = (): React.ReactElement => {
+	const countryCodes = ["US", "DE", "FR", "RU"] as CountryCode[];
```

Comment:
don't cast here, just manually define the type of `countryCodes` => `const countryCodes: CountryCode[] = [...]`